smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftWriter.kt

```diff
@@ -90,6 +90,7 @@ class SwiftWriter(private val fullPackageName: String, swiftImportContainer: Swi
                         sectionWriter.write(codeWriter, previousValue)
```

Comment:
Since this is in an `integrations.forEach { ... }` loop, I presume that if multiple integrations have notices, then the last one to be iterated will be the one that is used and the others are ignored?

Reply:
Yes, that is correct; only one should be setting copyright notice.

Reply:
Re: offline comment on whether `SwiftIntegration` is the right mechanism for this, I think it should be fine. Just need to ensure only one integration overrides the `copyrightNotice` method.

Reply:
I'll include that in the doc comment for the method.

Reply:
Changed to use `SwiftSettings` instead.